modules/FindBasisUniversal.cmake

```diff
@@ -119,96 +77,163 @@ endmacro()
 foreach(_component ${BasisUniversal_FIND_COMPONENTS})
     if(_component STREQUAL "Encoder")
         if(NOT TARGET BasisUniversal::Encoder)
-            find_path(BasisUniversalEncoder_DIR NAMES basisu_frontend.cpp
-                HINTS "${BASIS_UNIVERSAL_DIR}" "${BASIS_UNIVERSAL_DIR}/encoder"
-                NO_CMAKE_FIND_ROOT_PATH)
-            if(BasisUniversalEncoder_DIR)
-                set(BasisUniversal_Encoder_FOUND TRUE)
+            # Try to find compiled library when installed through vcpkg
+            find_library(BasisUniversalEncoder_LIBRARY_RELEASE basisu_encoder)
+            find_library(BasisUniversalEncoder_LIBRARY_DEBUG basisu_encoder HINTS "debug")
+
+            if(BasisUniversalEncoder_LIBRARY_DEBUG AND BasisUniversalEncoder_LIBRARY_RELEASE)
```

Comment:
`OR` here, and then wrap the `set_property()` for the libraries below in corresponding `if()`s